### PR TITLE
Add file list to formPreview

### PR DESCRIPTION
**Pull request by sd-92**

### CODE REVIEW

File: blocks/core/ff_module/ff_module-form-preview/ff_module-form-preview.js [hunk 1]
```diff
@@ -14,6 +15,7 @@ module.exports = React.createClass({
 				value: React.PropTypes.string,
 				modifier: React.PropTypes.string,
 				previewFor: React.PropTypes.string,
+				fileList: React.PropTypes.object,
```
Comment (dkhuntrods) [inline blocks/core/ff_module/ff_module-form-preview/ff_module-form-preview.js]: Might be a good idea to point to the propTypes definition in the FileList component itself here (with a `shape` wrapper) but not strictly necessary

**Comment by dkhuntrods** [comment]
One small comment; otherwise lgtm

**Comment by sd-92** [comment]
@dkhuntrods I've made the change :) if it looks good I'll go ahead and merge!

File: blocks/core/ff_module/ff_module-form-preview/ff_module-form-preview.js [hunk 2]
```diff
@@ -14,6 +15,9 @@ module.exports = React.createClass({
 				value: React.PropTypes.string,
 				modifier: React.PropTypes.string,
 				previewFor: React.PropTypes.string,
+				fileList: React.PropTypes.shape({ 
+					files: React.PropTypes.array(FileList.propTypes.files)
```
Comment (dkhuntrods) [inline blocks/core/ff_module/ff_module-form-preview/ff_module-form-preview.js]: Does `...shape(FileList.propTypes)` not work?

**Comment by sd-92** [comment]
@dkhuntrods You're right sorry, that works and looks less messy!

**Comment by dkhuntrods** [comment]
@Simonedeluca great! lgtm then :)

**Pull request merged**